noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/tagTweet.scala

```diff
@@ -0,0 +1,35 @@
+package edu.uci.ics.cloudberry.noah.feed
+
+import java.io.File
+
+import edu.uci.ics.cloudberry.gnosis._
+import edu.uci.ics.cloudberry.noah.adm.Tweet
+import edu.uci.ics.cloudberry.util.Profile._
+import twitter4j.TwitterObjectFactory
+
+import scala.collection.mutable
+
+object tagTweet {
+  val shapeMap = mutable.Map.empty[TypeLevel, String]
```

Comment:
a general principle is to use immutable structures as many as possible. e.g. the above code can be written as 
```
var shapeMap = Seq( StateLevel -> "neo/public/data/state.json", 
                                   CountyLevel -> "neo/public/data/county.json",
                                   CityLevel -> "neo/public/data/city.json").toMap
```